<?xml version="1.0"?>
<robot name="bonirob_simple" xmlns:xacro="http://ros.org/wiki/xacro">

  <xacro:property name="scale_factor" value="0.8"/>
  <xacro:property name="base_x" value="2.4"/>
  <xacro:property name="base_y" value="2"/>
  <xacro:property name="base_z" value="1"/>
  <xacro:property name="wheel_pad_x" value="0.3"/>
  <xacro:property name="wheel_pad_y" value="0.2"/>
  <xacro:property name="wheel_pad_z" value="0.2"/>
  <xacro:property name="wheel_r" value="0.25"/>
  <xacro:property name="wheel_l" value="0.2"/>

  <xacro:property name="wheel_pad_clearance" value="0.1"/>
  <xacro:property name="ground_clearance" value="${2*wheel_r + wheel_pad_clearance + 2*wheel_pad_z}"/>
  <xacro:property name="init_center_height" value="${ground_clearance + base_z/2}"/>

  <xacro:property name="base_mass" value="60" />
  <xacro:property name="wheel_pad_mass" value="5" />
  <xacro:property name="wheel_mass" value="5" />


  <xacro:property name="wheel_pad_origin_z" value="${-base_z/2 -1.5*wheel_pad_z}"/>
  <xacro:property name="wheel_origin_z" value="${-wheel_pad_z/2 -wheel_r -wheel_pad_clearance}"/>

  <xacro:property name="lidar_x" value="1" />
  <xacro:property name="lidar_pitch" value="0.7854" />
</robot>